gnomad_constraint/utils/generic.py

```diff
@@ -244,3 +244,34 @@ def remove_unnecessary_variants(
         exome_crit &= exome_ht.freq[freq_index].AF <= af_cutoff
     exome_ht = exome_ht.filter(exome_crit)
     return context_ht, exome_ht
+
+
+def get_all_pop_lengths(
+    ht, pops: Tuple[str], prefix: str = "observed_", skip_assertion: bool = False
+):
+    """
+    Get the minimum array length for specific per population annotations in `ht`.
```

Comment:
it might be helpful to make this description more informative -- this function is getting the minimum length of the downsampling counts array, right? why do we need this?